pkg/login/social/social.go

```diff
@@ -461,3 +465,23 @@ func (ss *SocialService) GetOAuthInfoProvider(name string) *OAuthInfo {
 func (ss *SocialService) GetOAuthInfoProviders() map[string]*OAuthInfo {
 	return ss.oAuthProvider
 }
+
+func (ss *SocialService) getUsageStats(ctx context.Context) (map[string]interface{}, error) {
```

Comment:
if usage is the generic name we have for most stats then we use it otherwise rename perhaps?
```suggestion
func (ss *SocialService) getSocialStats(ctx context.Context) (map[string]interface{}, error) {
```